daft/sql/sql_scan.py

```diff
@@ -50,7 +66,8 @@ def schema(self) -> Schema:
         return self._schema
 
     def display_name(self) -> str:
-        return f"SQLScanOperator(sql={self.sql}, url={self.url})"
+        conn_str = self.conn if isinstance(self.conn, str) else self.conn.__name__
```

Comment:
I think it's fine to use the repr for the connection, since it will be a function